perllib/FixMyStreet/Script/UpdateAllReports.pm

```diff
@@ -66,6 +67,21 @@ sub generate {
                 $open{$cobrand}{$body}{$type}++;
             }
         }
+
+        if ( FixMyStreet->config('AREAS_IN_ALLREPORTS_JSON') ) {
```

Comment:
If this is temporary (perhaps even if not), rather than a config variable, perhaps a command line argument to the update-all-reports script, saying you wish to include area data?